src/objects/tests/v1/test_filters.py

```diff
@@ -328,6 +328,31 @@ def test_filter_exclude_old_records(self):
         data = response.json()
         self.assertEqual(len(data), 0)
 
+    def test_filter_in_string(self):
+        record = ObjectRecordFactory.create(
+            data={"name": "demo1"}, object__object_type=self.object_type
+        )
+        record2 = ObjectRecordFactory.create(
+            data={"name": "demo2"}, object__object_type=self.object_type
+        )
+        ObjectRecordFactory.create(
+            data={"name": "demo3"}, object__object_type=self.object_type
+        )
+
+        response = self.client.get(self.url, {"data_attrs": "name__in__demo1|demo2"})
```

Comment:
Don't really like this format, but I cannot think of a better one. The comma is already used for multiple attributes.

Reply:
Hmm yeah, not sure what alternatives there are